packages/usa-in-page-navigation/src/test/in-page-navigation.spec.js

```diff
@@ -175,6 +180,40 @@ tests.forEach(({ name, selector: containerSelector }) => {
       assert(window.scroll.calledOnceWith(sinon.match({ top: 880 })));
     });
 
+    describe("Minimum heading count tests", () => {
+      beforeEach(() => {
+        body.innerHTML = TEMPLATE;
+        theNav = document.querySelector(THE_NAV);
+      });
+
+      afterEach(() => {
+        body.innerHTML = "";
+        window.location.hash = "";
+      });
+
+      it("does not render the in-page navigation when minimum heading count is too high", () => {
```

Comment:
**question**: What's the max number of headings?

Reply:
I am rewording this to say "does not render the in-page navigation when the default minimum heading count of 3 is reached" since template.html does not override the default value.

Reply:
Looking at this test, this part is testing to make sure the in-page nav is hidden when the number of headings is lower than what is defined as the minimum heading count. I don't believe there is any "max number" of headings that could be generated at this time.

Reply:
Tests were updated in [fdbd228](https://github.com/uswds/uswds/pull/6205/commits/fdbd2281ac74231198d9b1521a0ce2f90427c250)